www/data/docs_sidebar.yml

```diff
@@ -4,15 +4,50 @@ sidebar_links:
   links:
   - title: Docs Home
     link: "/docs"
-  - title: Using Habitat
+  - title: Install Habitat
+    link: "/docs/install-habitat"
+    sub_links:
+    - title: Download the Habitat CLI
+      link: "/docs/install-habitat/#install-habitat"
+    - title: Configure Your Workstation
+      link: "/docs/install-habitat/#configure-workstation"
+    - title: Download and Install FAQ
+      link: "/docs/install-habitat/#install-faq"
+  - title: Developing Packages
+    link: "/docs/developing-packages"
+    sub_links:
+    - title: Writing a Plan.sh
```

Comment:
Maybe this should be `Plan File` or just `Plan`? Only because there are also `plan.ps1s`, and a plan consists of more than just that file. What do you think?